client/components/pages/WelcomePage/WelcomePage.js

```diff
@@ -1,31 +1,46 @@
-import React, { useEffect } from 'react';
-import { useDispatch, useSelector } from 'react-redux';
-import { push } from 'connected-react-router';
-import R from 'ramda';
-
-import Section from 'react-bulma-companion/lib/Section';
-import Container from 'react-bulma-companion/lib/Container';
-import Title from 'react-bulma-companion/lib/Title';
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { push } from "connected-react-router";
+import R from "ramda";
+import request from "superagent";
+import Section from "react-bulma-companion/lib/Section";
+import Container from "react-bulma-companion/lib/Container";
+import Title from "react-bulma-companion/lib/Title";
+import Button from "react-bulma-companion/lib/Button";
 
 export default function WelcomePage() {
   const dispatch = useDispatch();
-  const { user } = useSelector(R.pick(['user']));
-
+  const { user } = useSelector(R.pick(["user"]));
+  const [response, setResponse] = React.useState("");
   useEffect(() => {
     if (!R.isEmpty(user)) {
-      dispatch(push('/home'));
+      dispatch(push("/home"));
     }
   }, []);
 
+  const handleAddTodo = () => {
```

Comment:
?

Reply:
It was when is play with the project in the example.. Don't apply this changes